docs/tutorials/introduction.ipynb

```diff
@@ -72,50 +77,60 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['alice'] = sc.Variable(dims=['row'], values=[1.0,1.1,1.2],\n",
-    "                         variances=[0.01,0.01,0.02], unit=sc.units.m)\n",
-    "sc.table(d)"
+    "ds['alice'] = sc.array(\n",
+    "    dims=['row'], values=[1.0, 1.1, 1.2], variances=[0.01, 0.01, 0.02], unit='m'\n",
+    ")\n",
+    "ds"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The column for `'alice'` contains two sub-columns with values and associated variances (uncertainties).\n",
+    "Under the hood, the column for `'alice'` contains two sub-columns with values and associated variances (uncertainties).\n",
     "The uncertainties are optional.\n",
+    "The datatype (`dtype`) is derived from the provided data, so passing `np.arange(3)` will yield a variable (column) containing 64-bit integers.\n",
```

Comment:
```suggestion
    "The datatype (`dtype`) is derived from the provided data, so passing a list of integers will yield a variable (column) containing 64-bit integers.\n",
```

I feel like saying `np.arange(3)` is explaining numpy's behaviour (granted that we are copying), but not directly related to the list of floats?